BetterThanRottenTomatoAPI/ReviewManagement/ReviewManagement.API/appsettings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "TomatoDb": "Server=DESKTOP-LQ9NL1I;Database=BetterThanRottenTomato;Trusted_Connection=True;"
```

Comment:
Don't need this, already a connection strings in the appsettings.development.json file